THIRD-PARTY

```diff
@@ -0,0 +1,1927 @@
+** Log4j-core; version 2.11.0 --
```

Comment:
QUES: Just curious. Is this file written or generated?

Reply:
Generated using an internal tool.